docs/reference/modules/ROOT/pages/http.adoc

```diff
@@ -1,6 +1,6 @@
 = HTTP
 
-Crux offers a REST API layer in the `crux-http-server` module that allows you to send transactions and run queries over HTTP.
+Crux offers a REST API layer in the `crux-http-server` module that allows you to send transactions and run queries over HTTP. (See the REST API reference xref:http-api.adoc[here])
```

Comment:
```
Crux offers a xref:...[REST API] layer in the `crux-http-server` module that allows you to send transactions and run queries over HTTP.
```
?

Also, think we talked about putting the REST API at the bottom of this file and linking to it?

Reply:
Ah, we did, and you have - think this is a broken link now?

Reply:
It is indeed - will sort!